themes/doom-ayu-light-theme.el

```diff
@@ -186,6 +186,11 @@ determine the exact padding."
    (ivy-minibuffer-match-face-3 :foreground common-accent :weight 'bold)
    (ivy-minibuffer-match-face-4 :foreground common-accent :weight 'bold)
 
+   ;;;; helm
+   (helm-ff-dotted-directory :inherit 'syntax-comment)
```

Comment:
What is syntax-comment? It doesn't exist in Emacs 28. Please don't inherit from faces that aren't guaranteed to exist when helm is loaded.